### PR TITLE
Update ds-paas image to ds

### PR DESCRIPTION
**Pull request by wstrange**
Update ds-paas image to ds
Quote slack url to prevent null expression
Fix istio gateway definition to use subdomain (not istio.subdomain)

Release 6.5.0 backport required? no
6.5.0 doc changes needed? no
7.0.0 doc changes needed? no

### CODE REVIEW

**Comment by wstrange** [comment]
The idm truststore is also removed - since it is just a copy of the default truststore that ships with idm/

File: helm/ds/templates/configmap.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ data:
   DJ_INSTANCE: {{ .Values.instance }}
```
Comment (markg58) [inline helm/ds/templates/configmap.yaml]: should this be quoted also?

File: helm/ds/values.yaml [hunk 2]
```diff
@@ -35,10 +35,10 @@ idmRepo:
   enabled: true
 
 image:
-  repository: forgerock-docker-public.bintray.io/forgerock/ds-paas
-  pullPolicy: IfNotPresent
-  #pullPolicy: Always
-  tag: 7.0.0-M134.2
+  repository: forgerock-docker-public.bintray.io/forgerock/ds
```
Comment (markg58) [inline helm/ds/values.yaml]: why is this changing?  Is /ds for snapshots and /ds-paas for milestones & release builds?

Comment (csovant) [inline helm/ds/values.yaml]: DS team does not provide anymore the ds-pass archive?

Comment (wstrange) [reply helm/ds/values.yaml]: No - Matt changed it to ds

File: helm/ds/templates/configmap.yaml (hunk 1 above)
Comment (wstrange) [reply helm/ds/templates/configmap.yaml]: probably. It theory it is never left blank - which is what caused the issue with the slack url

File: helm/ds/values.yaml (hunk 2 above)
Comment (wstrange) [reply helm/ds/values.yaml]: ds-paas is gone
it is just ds now

Comment (csovant) [reply helm/ds/values.yaml]: Ok thanks for the info

Comment (markg58) [reply helm/ds/values.yaml]: :thumbsup:

**Pull request merged**